inc/templates/search-replace.php

```diff
@@ -22,8 +22,11 @@
 		</tr>
 		<tr>
 			<th><label for="csv"><strong><?php esc_html_e( 'CSV Format Search/Replace:', 'search-and-replace'); ?></strong></label></th>
-			<td><textarea id="csv" cols="62" name="csv" placeholder="search value, replace value (one per line)"><?php $this->get_csv_value() ?></textarea></td>
-		</tr>
+			<td>
+                            <textarea id="csv" cols="46"  rows="5" name="csv" placeholder="<?php esc_html_e("search value, replace value (one per line)" , "search-and-replace")?>" ><?php $this->get_csv_value() ?></textarea>
```

Comment:
a hint again. We need singel quotes instead then double quotes and also a space before quotes, like `( '...`. Can you change this. Thanks.

Reply:
@bueltge  thanks, please check my commit again :)